herbert_core/utilities/classes/@unique_objects_container/private/remove_noncomplying_members_.m

```diff
@@ -0,0 +1,37 @@
+function self = remove_noncomplying_members_(self,new_class_name)
+% Remove all elements, which are not the members of the class of type
+% specified as input.
```

Comment:
why not? It runs at the check of setting container type. if you have mixed container and want to set type on it, this will successfully and without big overhead do that.  Not very important in current code but can be used for number of other situations.